src/ImageSharp/Common/Helpers/Vector128Utilities.cs

```diff
@@ -32,10 +35,10 @@ public static bool SupportsShuffleFloat
     /// <summary>
     /// Gets a value indicating whether shuffle operations are supported.
     /// </summary>
-    public static bool SupportsShuffleByte
+    public static bool SupportsShuffleNativeByte
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        get => Ssse3.IsSupported || AdvSimd.Arm64.IsSupported;
+        get => Ssse3.IsSupported || AdvSimd.Arm64.IsSupported || PackedSimd.IsSupported;
```

Comment:
For Arm64 and WASM you should just be able to use (for `byte`) `Vector128.Shuffle` due to how `VectorTableLookup` and `Swizzle` work.

You really just want at least `Ssse3` for x86/x64, since they cannot be done otherwise.

So perhaps you want:
```csharp
get
{
    if (Vector128.IsHardwareAccelerated)
    {
        if (RuntimeInformation.ProcessArchitecture is Architecture.X86 or Architecture.X64)
        {
            return Ssse3.IsSupported;
        }

        // You could optionally do:
        //    return ProcessArchitecture is Architecture.Arm64 or Architecture.Wasm;
        // if you wanted to restrict it to platforms you know should be safe
        return true;
    }

    return false;
}
```

Reply:
Good catch. I’ll review my other helpers.